src/components/ContextBar.js

```diff
@@ -0,0 +1,66 @@
+import { Fragment } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { Clickable, comingSoon, makeMenuIcon, MenuButton } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import PopupTrigger from 'src/components/PopupTrigger'
+import colors, { terraSpecial } from 'src/libs/colors'
+
+
+const contextBarStyles = {
+  contextBarContainer: {
+    display: 'flex', flexWrap: 'wrap'
+  },
+  contextBarButton: {
+    display: 'flex', //, justifyContent: 'center', alignItems: 'center',
+    borderBottom: `1px solid ${colors.accent()}`,
+    padding: '1rem 1rem 1rem 1rem',
+    color: colors.accent(),
+    backgroundColor: colors.accent(0.2)
+  }
+}
+
+export const ContextBarButtons = ({ setDeletingWorkspace, setCloningWorkspace, setSharingWorkspace, isOwner, canShare }) => {
+  return div({ style: contextBarStyles.contextBarContainer }, [
+    h(PopupTrigger, {
```

Comment:
Might I recommend the new `MenuTrigger`? It will get all the accessibility/tab movement pieces for free!